modules/retail/pages/retail-components.adoc

```diff
@@ -93,7 +87,16 @@ Then all devices of a particular type can be managed as one.
 
 
 
-[[retail.sect.components.fitting.formulas]]
+=== System groups
```

Comment:
```suggestion
=== System Groups
```

Reply:
Maybe even call it Branch System Groups.
Because hardware types also uses system groups, but they are mentioned above and this paragraph is solely about system groups named after branch id and used for per-branch management.